elements/lisk-client/index.html

```diff
@@ -8,6 +8,6 @@
 		Open the console to get started with Lisk Client.
 		<div id="output"></div>
 
-		<script src="dist-browser/index.min.js"></script>
+		<script src="dist-browser/index.js"></script>
```

Comment:
any specific reason for not using `min`?

Reply:
This file is for testing and i wanted to use unuglify version for test